src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[figureSupplier.getMaxNumber()];
```

Comment:
```suggestion
        Figure[] figures = new Figure[ARRAY_SIZE];
```

create constant ARRAY_SIZE